src/groovy/org/pih/warehouse/core/ActivityCode.groovy

```diff
@@ -93,6 +94,10 @@ enum ActivityCode {
         ]
     }
 
+    static listWithAutosave() {
```

Comment:
Instead of this check if the autosave activity code is enabled in the `list()` as:
```groovy
static list() {
    def activityCodes = [
            MANAGE_INVENTORY,
            ADJUST_INVENTORY,
            APPROVE_ORDER,
            APPROVE_REQUEST,
            PLACE_ORDER,
            PLACE_REQUEST,
            FULFILL_ORDER,
            FULFILL_REQUEST,
            SEND_STOCK,
            RECEIVE_STOCK,
            CONSUME_STOCK,
            CROSS_DOCKING,
            PUTAWAY_STOCK,
            PICK_STOCK,
            EXTERNAL,
            ENABLE_NOTIFICATIONS,
            PACK_SHIPMENT,
            PARTIAL_RECEIVING,
            REQUIRE_ACCOUNTING,
            ENABLE_CENTRAL_PURCHASING,
            HOLD_STOCK,
            SUBMIT_REQUEST,
            DYNAMIC_CREATION
    ]

    return CH.config.openboxes.autosave.enabled ? activityCodes + [AUTOSAVE] : activityCodes
}
```